index.bs

```diff
@@ -44,9 +71,18 @@ spec: SHA2; urlPrefix: https://csrc.nist.gov/publications/fips/fips180-4/fips-18
     text: SHA-256; url: #
     text: SHA-384; url: #
     text: SHA-512; url: #
+
 spec: RFC8288; urlPrefix: https://tools.ietf.org/html/rfc8288
   type: http-header
     text: link
+
+spec: RFC9651; urlPrefix: https://tools.ietf.org/html/rfc9651
+  type: dfn
+    text: Dictionary; url: name-dictionaries
+    text: inner list; url: name-inner-lists
+    text: token; url: name-tokens
+    text: parsing a dictionary; url: name-parsing-a-dictionary
```

Comment:
This should probably use https://fetch.spec.whatwg.org/#concept-header-list-get-structured-header instead. There's no reason for specifications outside of Fetch to directly interface with this RFC for parsing.